backend/src/handlers/user_handler.go

```diff
@@ -290,3 +292,64 @@ func validateUser(user *models.User) string {
 	}
 	return ""
 }
+
+func (srv *Server) handleResidentVerification(w http.ResponseWriter, r *http.Request, log sLog) error {
+	args := srv.getQueryContext(r)
+	transferFacilityId, err := strconv.Atoi(r.URL.Query().Get("facility_id"))
+	if err != nil {
+		return newInvalidIdServiceError(err, "facility ID")
+	}
+	log.add("trans_facility_id", transferFacilityId)
+	userID, err := strconv.Atoi(r.URL.Query().Get("user_id"))
+	if err != nil {
+		return newInvalidIdServiceError(err, "user ID")
+	}
+	log.add("user_id", userID)
+	docID := r.URL.Query().Get("doc_id")
+	user, err := srv.Db.GetUserByDocIDAndID(&args, docID, userID)
+	if err != nil {
+		return writeJsonResponse(w, http.StatusNotFound, "Resident not found")
+	}
+	log.add("doc_id", docID)
+	programNames, err := srv.Db.GetTransferProgramConflicts(&args, user.ID, transferFacilityId)
+	if err != nil {
+		return writeJsonResponse(w, http.StatusNotFound, "Resident not found")
+	}
+	validResident := struct {
```

Comment:
I understand what this represents but `validResident` sticks out to me as a name that isn't as clear. I was thinking `transferRecord` but that seems like it would be used after its complete. And `transferRequest` is probably a no because it would be conflating it with the HTTP request.

Reply:
I  saw `transRequest` down below and I don't feel like it was much of an issue. Maybe we use `transferNotes` here?...